python/test/t_MaximumLikelihoodFactory_std.py

```diff
@@ -15,3 +15,45 @@
 inf_distribution = factory.build(sample)
 print(inf_distribution)
 
+class TruncatedFactory(object):
+    def __init__(self, distribution, lb=None, ub=None):
```

Comment:
@sofianehaddad yes, but there's no generic way of knowing which parameters should be in bounds